shippable.yml

```diff
@@ -20,7 +20,7 @@ build:
     - cp -f ./app.client/config/config.$TENANT.ts ./app.client/config/config.ts
     - cp -f ./app.client/styles/config.$TENANT.ts ./app.client/styles/config.ts
     - cp -f ./app.client/images/favicon.$TENANT.png ./app.client/images/favicon.png
-    - cp -f ./app.server/config/config.$TENANT.ts ./app.server/config/config.ts
+    #- cp -f ./app.server/config/config.$TENANT.ts ./app.server/config/config.ts
```

Comment:
This must be related to the comment above - let's not just comment something out without explanation - the config just needs to go in the correct file.

Reply:
Yes but at dev time its a nightmare to deal with

Reply:
This is the same issue as https://github.com/jncc/deli/pull/48#discussion_r258578535 so I'll resolve